lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -3845,15 +3843,34 @@ ThreadSafeASTContext SwiftASTContext::GetASTContext() {
     m_ast_context_up->addModuleLoader(std::move(memory_buffer_loader_up));
   }
 
+  // 2. Create the explicit swift module loader.
+  if (props.GetUseSwiftExplicitModuleLoader()) {
+    auto &search_path_opts = GetCompilerInvocation().getSearchPathOptions();
+    std::unique_ptr<swift::ModuleLoader> esml_up =
+        swift::ExplicitSwiftModuleLoader::create(
```

Comment:
Does this loader load module from CAS correctly?

So for caching build, we actually stored the explicit module map inside the CAS so the file is actually loaded from there. We can probably switch that up to use a regular file instead now (I created a special code path because the explicit module map wasn't deterministic before, but it should be now).

Reply:
My thinking was that this behaves exactly the same as loading a module dependency of an explicit module in the compiler.